docs/contribution.rst

```diff
@@ -0,0 +1,128 @@
+======================
+Contributing to ecl2df
+======================
+
+Contributing to ecl2df is easiest on Linux computers. Windows has not been
+tested, and for Mac you will have to compile OPM yourself.
+
+Getting started as a developer
+------------------------------
+
+Clone the master repository from Github::
+
+  git clone https://github.com/equinor/ecl2df
+
+After cloning, change to the newly created ``ecl2df`` directory. The commands
+below assume you are in this directory.
+
+It is recommended that you also fork the repository on Github, in order to push
+your branches to your fork. After forking, add the fork to your cloned copy::
+
+  git remote add fork git@github.com:<youruser>/ecl2df
+
+This requires a valid login setup with SSH keys for you github account, needed
+for write access.
+
+After cloning, you should make a Python virtual environment in which you install
+ecl2df and its dependencies. If you want to create a new virtual environment for
+ecl2df, you can do something like the following::
+
+  virtualenv venv-ecl2df
+  source venv-ecl2df/bin/activate
```

Comment:
virtualenv is python2?

mkdir py3.6-venv-ecl2df; python -m venv py3.6-venv-ecl2df   (I prefer to have python version included in name but up to you)

Reply:
Thanks, virtualenv points to py2 still on ubuntu 20.04.